src/messages/variant.rs

```diff
@@ -31,13 +32,15 @@ use xor_name::XorName;
 /// Message variant
 pub(crate) enum Variant {
     /// Inform other sections about our section or vice-versa.
-    OtherSection {
-        /// `SectionAuthorityProvider` of the sender's section, with the proof chain.
-        section_auth: Proven<SectionAuthorityProvider>,
-        /// Nonce that is derived from the incoming message that triggered sending this
-        /// message. It's purpose is to make sure that `OtherSection`s that are identical
-        /// but triggered by different messages are not filtered out.
-        nonce: MessageHash,
+    SectionKnowledge {
+        /// `SectionAuthorityProvider` and `SectionChain` of the sender's section, with the proof chain.
+        src_info: (Proven<SectionAuthorityProvider>, SectionChain),
```

Comment:
Why do we need it to be a tuple rather than having two separate fields?

Reply:
@Yoga07 ^^

Reply:
No particular reason there :) Just thought it would be better to bundle them as `src_info`. Can split it up 👍